src/main/java/entities/Customer.java

```diff
@@ -1,11 +1,24 @@
 package entities;
 
 public class Customer {
+
     private int customerId;
     private boolean isAdmin;
     private String customerName;
-    private String LastName;
-    private String personId;
+    private String customerLastName;
+    private String customerPersonId;
```

Comment:
what is personId and how will we use it?

Reply:
PersonID - this is the passport data

Reply:
ok then

Reply:
such as 40 08 131-33-4444